index.d.ts

```diff
@@ -1,14 +1,16 @@
 export {EventEmitter as default};
 
+export type Listener<Params extends unknown[] = []> = (...params: Params) => void
+
```

Comment:
This type is the replacement for `ListenerFn`, right?

Reply:
Yes